docs/news/news-v5.0.x.rst

```diff
@@ -11,254 +11,335 @@ Open MPI version 5.0.0rc12
+   v5.0.0.
 
-   The new PRRTE based runtime environment supports PMIx-tools API
-   instead of the legacy MPIR API for debugging parallel jobs.
+   Open MPI now uses the `PRRTE <https://github.com/openpmix/prrte>`_
+   runtime environment, which supports the `PMIx <https://pmix.org/>`_
+   tools API |mdash| instead of the legacy MPIR API |mdash| for
+   debugging parallel jobs.
 
-   See https://github.com/hpc/mpir-to-pmix-guide for more
-   information.
+   Users who still need legacy MPIR support should see
+   https://github.com/hpc/mpir-to-pmix-guide for more information.
 
-.. admonition:: Zlib is suggested for better user experience
+.. admonition:: Zlib is suggested for better performance
    :class: note
 
-   PMIx will optionally use `Zlib <https://github.com/madler/zlib>`_
-   to compress large data streams.  This may result in faster startup
-   times and smaller memory footprints (compared to not using
-   compression).  The Open MPI community recommends building Zlib
-   support with PMIx, regardless of whether you are using an
-   externally-installed PMIx or the PMIx that is installed with Open
-   MPI.
-
-.. caution::
-   Open MPI no longer builds 3rd-party packages
-   such as Libevent, HWLOC, PMIx, and PRRTE as MCA components
-   and instead:
-
-   #. Relies on external libraries whenever possible, and
-   #. Builds the 3rd party libraries only if needed, and as independent
-      libraries, rather than linked into the Open MPI core libraries.
-
+   `PMIx <https://pmix.org/>`_ will optionally use `Zlib
+   <https://github.com/madler/zlib>`_ to compress large data streams.
+   This may result in faster startup times and smaller memory
+   footprints (compared to not using compression).
+
+   The Open MPI community recommends building PMIx with Zlib support,
+   regardless of whether you are using an externally-installed PMIx or
+   the bundled PMIx that is included with Open MPI distribution
+   tarballs.
+
+.. caution:: Open MPI has changed the default behavior of how it
+             builds and links against its :ref:`required 3rd-party
+             packages <label-install-required-support-libraries>`:
+             `Libevent <https://libevent.org/>`_, `Hardware Locality
+             <https://www.open-mpi.org/projects/hwloc/>`_, `PMIx
+             <https://pmix.org/>`_, and `PRRTE
+             <https://github.com/openpmix/prrte>`_.
+
+             #. For each of the libraries, if an external installation
+                of that library can be found, it will be used.
+             #. Otherwise, an internally-bundled version of that
+                library will be compiled and installed.  Different
+                compared to previous releases of Open MPI, the library
+                will be installed into the Open MPI installation
+                prefix, as opposed to being "hidden" by virtue of
+                being statically linked into Open MPI components.
```

Comment:
Hey @bwbarrett I'm not entirely happy with how this section turned out.  Do you have any suggestions?

(you can see how it renders here: https://ompi--11707.org.readthedocs.build/en/11707/news/news-v5.0.x.html -- it's the "Caution" block right near the top of the page)

Reply:
The last bullet I think reads weird because it's really saying two different things.  What about:

Open MPI has changed the default behavior of how it builds and links against its [required 3rd-party packages](https://ompi--11707.org.readthedocs.build/en/11707/installing-open-mpi/required-support-libraries.html#label-install-required-support-libraries): [Libevent](https://libevent.org/), [Hardware Locality](https://www.open-mpi.org/projects/hwloc/), [PMIx](https://pmix.org/), and [PRRTE](https://github.com/openpmix/prrte).

* Unlike previous versions of Open MPI, Open MPI 5.0 and later will prefer an external package that meets our version requirements, even if it is older than our internal version.

* To simplify managing dependencies, any required packages that Open MPI 5.0 bundles will be installed in Open MPI's installation prefix, without name mangling.  For example, if libevent cannot be found and Open MPI builds its bundled version, a libevent.so will be installed in Open MPI's installation tree.  This is different from previous releases, where Open MPI name mangled the library symbols and then pulled the library into libmpi.so.